provisioner/inspec/provisioner.go

```diff
@@ -334,11 +347,14 @@ func (p *Provisioner) executeInspec(ui packer.Ui, comm packer.Communicator, priv
 	args = append(args, "--backend", p.config.Backend)
 	args = append(args, "--host", p.config.Host)
 
+	if p.config.User != "" {
+		args = append(args, "--user", p.config.User)
+	}
```

Comment:
I saw some winrm backend setting the User config, so I guess this arg can be set when this field is set.